articles/hdinsight/interactive-query/apache-interactive-query-get-started.md

```diff
@@ -25,6 +25,9 @@ You can access the Hive service in the Interactive Query cluster only via Apache
 
 For information about creating a HDInsight cluster, see [Create Apache Hadoop clusters in HDInsight](../hdinsight-hadoop-provision-linux-clusters.md). Choose the Interactive Query cluster type.
 
+>[!Important]
+>The minimum headnode size for Interactive Query clusters is Standard_D13_v2. See the (Azure VM Sizing Chart)[https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-sizes-specs#dv2-series] for more information.

```

Comment:
Hi Taylor, 
Two things:
1. We have to remove en-us from all links - they must be language agnostic. 

Please use a relative link instead of http address, so that the address is verified at build time. 
`../../cloud-services/cloud-services-sizes-specs.md#dv2-series`

2. Please add a space after each bracket. For example
```
> [!IMPORTANT]
> Essential information required for user success.
```